components/brave_ads/core/internal/account/confirmations/user_data_builder/confirmation_user_data_builder.cc

```diff
@@ -24,12 +23,8 @@ void BuildFixedUserDataCallback(base::Value::Dict user_data,
   UserDataInfo confirmation_user_data;
 
   confirmation_user_data.dynamic = std::move(dynamic_user_data);
-
   confirmation_user_data.fixed = std::move(fixed_user_data);
-
-  if (UserHasJoinedBraveRewards()) {
-    confirmation_user_data.fixed.Merge(std::move(user_data));
-  }
+  confirmation_user_data.fixed.Merge(std::move(user_data));
```

Comment:
This is data passed as part of `DepositWithUserData` and will be guarded on a per use case.